docs/src/lateral/local-inertial.md

```diff
@@ -74,23 +74,75 @@ fixed at 0.0 m.
 Simplified reservoirs and lakes models can be included as part of the local inertial model, 
 see also [Reservoirs and Lakes](@ref).
 
+## Overland flow (2D)
+For the simulation of 2D overland flow on a staggered grid the numerical scheme proposed by
+de Almeida et al. (2012) is adopted. The explicit solution for the estimation of water
+discharge between two cells in the x-direction is of the following form (following the
+notation of Almeida et al. (2012)):
+
+```math
+Q_{i-1/2}^{n+1} = \frac{\left[ \theta Q_{i-1/2}^{n} +\frac{(1-\theta)}{2}(Q_{(i-3/2)}^{n} + \\
+  Q_{(i+1/2)}^{n})\right]- g h_f \frac{\Delta t}{\Delta x} (y^n_i - y^n_{i-1}) \Delta y}{1+g\Delta t \\
+   n^2 |Q_{i-1/2}^{n}|/(h_f^{7/3} \Delta y)}
+```
+
+where subscripts ``i`` and ``n`` refer to space and time indices, respectively. Subscript
+``i-1/2`` is to the link between node ``i`` and ``i-1``, subscript ``i+1/2`` is the link
+between node ``i`` and node ``i+1``, and subscript ``i-3/2`` is the link between node ``i``
```

Comment:
The difference between subscript ``i-3/2`` and subscript ``i-1/2`` is not clear from this text. In the code it is ``qu`` and ``qd``, perhaps include this here as well?

Reply:
Thanks for spotting this. The link `i-3/2` was not described correctly (should be between node `i-1` and `i-2`). Also added an image of the numerical scheme.